cmd/sti/main.go

```diff
@@ -92,6 +92,24 @@ func newCmdBuild(req *api.Request) *cobra.Command {
 	return buildCmd
 }
 
+func newCmdCreate() *cobra.Command {
+	return &cobra.Command{
+		Use:   "create <image> <source>",
```

Comment:
Change:
* image -> imageName
* source -> [destination] directory
Improve long descriptions with what is what.